src/App.jsx

```diff
@@ -14,33 +19,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState(DEFAULT_SORT_FIELD);
+  const [reversed, setReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  let visibileGoods = [...goodsFromServer];
```

Comment:
There is a typo in the variable name `visibileGoods`. It should be `visibleGoods`.